webclient/dokument/views.py

```diff
@@ -927,7 +935,7 @@ def odpojit(request, ident_doku, ident_zaznamu, view):
         return JsonResponse({"redirect": reverse(f"{view}:detail")}, status=404)
     if len(relace_dokumentu) == 1:
         orphan_dokument = relace_dokumentu[0].dokument
-        if "X-" in orphan_dokument.ident_cely:
+        if orphan_dokument.stav == D_STAV_ARCHIVOVANY:
```

Comment:
@jnihnat Nejsem si jistý, že tato náhrada je správně. Cílem je mazat pouze dokumenty, které ještě ani jednou neprošly archivací, tj. ty s "X-" v ident_cely. Možná se to řeší jinde v kódu, ale pokud ne, pak je tato záměna chybně.

Reply:
opraveno. Poprosim do budoucna lip popstav v pozadavku, tam byl problem ze se mazali i archivovane dokumenty. ne vzdy je pak jasne jak presne to ma fungovat.